tests/kola/version/rhel-major-version

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# kola: { "exclusive": false }
+set -xeuo pipefail
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+# checks to ensure that the only packages from the RHEL major version are included
+
+source /etc/os-release 
+var=$(echo "${RHEL_VERSION}" | cut -d. -f1)
+    for x in $(rpm -qa --queryformat='%{RELEASE}\n' | grep -oP 'el\s*\K\d+');
+        do
+            if [ "$var" -ne $((x)) ] 
```

Comment:
Double brackets `[[` are preferred. You can also place the `then` on the same line after a `;` here.